### PR TITLE
Personal info tab

### PR DESCRIPTION
**Pull request by LuciusA**

### CODE REVIEW

File: src/components/user/ChangePasswordModal.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import {
+  Modal,
+  Backdrop,
+  Fade,
+  TextField,
+  Button,
+  makeStyles
+} from "@material-ui/core";
+
```
Comment (ZzZimm) [inline src/components/user/ChangePasswordModal.tsx:10]: Hello

File: src/components/user/ChangePasswordModal.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      className={styles.changePasswordModalLayout__modal}
+      open={open}
+      onClose={handleClose}
+      closeAfterTransition
+      BackdropComponent={Backdrop}
+      BackdropProps={{
+        timeout: 500
+      }}
+    >
+      <Fade in={open}>
```
Comment (ComeTiss) [inline src/components/user/ChangePasswordModal.tsx:63]: We need to ask for previous password too

File: src/components/user/PersonalProfileDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    confirmPassword: "",
+    lastName: user.lastName,
+    firstName: user.firstName,
+    civility: "",
+    functionTitle: user.functionTitle,
+    emailDisplay: user.email,
+    phoneNumber: "",
+    whatsAppNumber: "",
+    weChatId: "",
+    skypeId: ""
```
Comment (ComeTiss) [inline src/components/user/PersonalProfileDetails.tsx:76]: We need to check with Diane for that - to me one contact field seems enough, in addition of phone & email. We can then define an enum and add a dropdown list

**Review by ComeTiss — DISMISSED** [review]

**Review by ComeTiss — CHANGES_REQUESTED** [review]
you need to require previous password to change password

**Review by ComeTiss — APPROVED** [review]
Nice!

File: src/components/user/PersonalProfileDetails.tsx (hunk 3 above)
Comment (LuciusA) [reply src/components/user/PersonalProfileDetails.tsx:76]: Ok

File: src/components/user/ChangePasswordModal.tsx (hunk 2 above)
Comment (LuciusA) [reply src/components/user/ChangePasswordModal.tsx:63]: Ok

**Pull request merged**